src/templates/customer/login.template.html

```diff
@@ -53,28 +53,28 @@ <h3>New Customer? Create an Account.</h3>
 							<div class="form-group">
 								<label for="reg_username">Username </label>
 								<span class="text-danger small">Required</span>
-								<input class="form-control" type="text" id="reg_username" name="reg_username" value="[%format type:'date' format:'#h#i#s'%]now[%END format%][%random_number length:'6'%][%end random_number%]" size="25" maxlength="20">
+								<input class="form-control" type="text" id="reg_username" name="reg_username" value="[%format type:'date' format:'#h#i#s'%]now[%END format%][%random_number length:'6'%][%end random_number%]" size="25" maxlength="20" required>
```

Comment:
This is hidden to the user so it shouldn't have a required attribute. But you should look into changing this to a type="hidden" input and placing it under the redirect input. The reg_username input is used to create a username and is made up of a random number. This form-group is unnecessary and is readable to screen readers, and if a customer want to customise the template to include the reg_username as an editable field then they can still easily do that.